lib/ReactViews/Workbench/Controls/ViewingControls.jsx

```diff
@@ -367,6 +367,8 @@ const ViewingControls = observer(
             {/* </If> */}
             {/* <If condition={item.showsInfo}> */}
             <WorkbenchButton
+              // to={`${CATALOG_ROUTE}${URI.encode(item.uniqueId)}`}
+              // type="button"
```

Comment:
? ?

Reply:
Yeah. Looks like "About data" is not working.

Reply:
Fixed via https://github.com/TerriaJS/terriajs/pull/4679/commits/649941e2da75f9a3544933f26e5762640a97f0c4